prodtests/full_system_test.sh

```diff
@@ -46,6 +46,7 @@ NHBPERTF=${NHBPERTF:-128}
 RUNFIRSTORBIT=${RUNFIRSTORBIT:-0}
 FIRSTSAMPLEDORBIT=${FIRSTSAMPLEDORBIT:-0}
 OBLIGATORYSOR=${OBLIGATORYSOR:-false}
+RUNNUMBER=300000 # a default un-anchored run number
```

Comment:
can you add to https://github.com/AliceO2Group/AliceO2/blob/e8ae9ffc0b8559a2001729b4b23511d65936a110/prodtests/full_system_test.sh#L78
`;HBFUtils.runNumber=${RUNNUMBER}`